spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt

```diff
@@ -179,6 +179,7 @@ internal object PaletteTokens {
     internal val Surfer80 = Color(0xFFD1D0D5)
     internal val Surfer90 = Color(0xFFEBEBED)
     internal val Surfer95 = Color(0xFFF8F8F9)
+    internal val Surfer975 = Color(0xFFF8F8F9)
```

Comment:
This seems to be a duplicate of `Surfer95 `

Reply:
Yep it's `Surfer95` that has the wrong name and should be deleted instead